### PR TITLE
merge Release/live service/55.1 into master

### PR DESCRIPTION
**Pull request by JessiePotter1**

### CODE REVIEW

File: DFC.App.Account.UnitTests/Controllers/ErrorControllerTests.cs [hunk 1]
```diff
@@ -23,13 +25,15 @@ public class ErrorControllerTests
...
+        //private IDocumentService<CmsApiSharedContentModel> _documentService;
         private IConfiguration _config;
+        private ISharedContentRedisInterface _sharedContentRedisInterface;
+
 
         [SetUp]
         public void Init()
         {
-            _documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
+            //_documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
```
Comment (hnandra) [inline DFC.App.Account.UnitTests/Controllers/ErrorControllerTests.cs:36]: Delete comment

File: DFC.App.Account.UnitTests/Controllers/HomeControllerTests.cs [hunk 2]
```diff
@@ -35,13 +37,14 @@ public class HomeControllerTests
         private IOptions<ActionPlansSettings> _actionPlansSettings;
         private IDssReader _dssReader;
         private HomeController _controller;
-        private IDocumentService<CmsApiSharedContentModel> _documentService;
+        //private IDocumentService<CmsApiSharedContentModel> _documentService;
```
Comment (hnandra) [inline DFC.App.Account.UnitTests/Controllers/HomeControllerTests.cs:40]: Delete comment

File: DFC.App.Account.UnitTests/Controllers/HomeControllerTests.cs [hunk 3]
```diff
@@ -35,13 +37,14 @@ public class HomeControllerTests
...
         private IConfiguration _config;
+        private ISharedContentRedisInterface _sharedContentRedisInterface;
 
         [SetUp]
         public void Init()
         {
-            _documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
+            //_documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
```
Comment (hnandra) [inline DFC.App.Account.UnitTests/Controllers/HomeControllerTests.cs:47]: Delete comment

File: DFC.App.Account.UnitTests/Controllers/SessionTimeoutControllerTests.cs [hunk 4]
```diff
@@ -22,13 +24,14 @@ public class SessionTimeoutControllerTests
         private IOptions<CompositeSettings> _compositeSettings;
         private IAuthService _authService;
         private IOptions<AuthSettings> _authSettings;
-        private IDocumentService<CmsApiSharedContentModel> _documentService;
+        //private IDocumentService<CmsApiSharedContentModel> _documentService;
```
Comment (hnandra) [inline DFC.App.Account.UnitTests/Controllers/SessionTimeoutControllerTests.cs:27]: Delete comment

File: DFC.App.Account.UnitTests/Controllers/SessionTimeoutControllerTests.cs [hunk 5]
```diff
@@ -22,13 +24,14 @@ public class SessionTimeoutControllerTests
...
         private IConfiguration _config;
-
+        private ISharedContentRedisInterface _sharedContentRedisInterface;
         [SetUp]
         public void Init()
         {
-            _documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
+            //_documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
```
Comment (hnandra) [inline DFC.App.Account.UnitTests/Controllers/SessionTimeoutControllerTests.cs:33]: Delete comment

File: DFC.App.Account.UnitTests/Controllers/ShcDeletedControllerTests.cs [hunk 6]
```diff
@@ -24,13 +26,13 @@ public class ShcDeletedControllerTests
...
         private ISkillsHealthCheckService _skillsHealthCheckService;
-        private IDocumentService<CmsApiSharedContentModel> _documentService;
         private IConfiguration _config;
+        private ISharedContentRedisInterface _sharedContentRedisInterface;
 
         [SetUp]
         public void Init()
         {
-            _documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
+            //_documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
```
Comment (hnandra) [inline DFC.App.Account.UnitTests/Controllers/ShcDeletedControllerTests.cs:35]: Delete comment

File: DFC.App.Account.UnitTests/Controllers/YourDetailsControllerTests.cs [hunk 7]
```diff
@@ -31,16 +33,18 @@ class YourDetailsControllerTests
         private YourDetailsController _controller;
         private Customer _customer;
         private IAuthService _authService;
-        private IDocumentService<CmsApiSharedContentModel> _documentService;
+       //private IDocumentService<CmsApiSharedContentModel> _documentService;
```
Comment (hnandra) [inline DFC.App.Account.UnitTests/Controllers/YourDetailsControllerTests.cs:36]: Delete comment

File: DFC.App.Account.UnitTests/Controllers/YourDetailsControllerTests.cs [hunk 8]
```diff
@@ -31,16 +33,18 @@ class YourDetailsControllerTests
...
-
+        private ISharedContentRedisInterface _sharedContentRedisInterface;
         [SetUp]
         public void Init()
         {
             _compositeSettings = Options.Create(new CompositeSettings());
             _dssService = Substitute.For<IDssReader>();
             _authService = Substitute.For<IAuthService>();
-            _documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
+            //_documentService = Substitute.For<IDocumentService<CmsApiSharedContentModel>>();
```
Comment (hnandra) [inline DFC.App.Account.UnitTests/Controllers/YourDetailsControllerTests.cs:45]: Delete comment

File: DFC.App.Account/Controllers/DeleteAccountController.cs [hunk 9]
```diff
@@ -7,19 +7,22 @@
 using Microsoft.Extensions.Options;
 using System.Threading.Tasks;
 using DFC.APP.Account.Data.Models;
-using DFC.Compui.Cosmos.Contracts;
+//using DFC.Compui.Cosmos.Contracts;
```
Comment (hnandra) [inline DFC.App.Account/Controllers/DeleteAccountController.cs:10]: Delete comment

File: DFC.App.Account/Controllers/SessionTimeoutController.cs [hunk 10]
```diff
@@ -3,20 +3,25 @@
 using DFC.App.Account.Models;
 using DFC.App.Account.Services;
 using DFC.App.Account.ViewModels;
-using DFC.Compui.Cosmos.Contracts;
+//using DFC.Compui.Cosmos.Contracts;
```
Comment (hnandra) [inline DFC.App.Account/Controllers/SessionTimeoutController.cs:6]: Delete comment

File: DFC.App.Account/Controllers/YourDetailsController.cs [hunk 11]
```diff
@@ -8,8 +8,9 @@
 using Microsoft.Extensions.Options;
 using System.Threading.Tasks;
 using DFC.APP.Account.Data.Models;
-using DFC.Compui.Cosmos.Contracts;
+//using DFC.Compui.Cosmos.Contracts;
```
Comment (hnandra) [inline DFC.App.Account/Controllers/YourDetailsController.cs:11]: Delete comment

**Review by hnandra — CHANGES_REQUESTED** [review]
Delete comments

**Review by hnandra — APPROVED** [review]

**Review by gcomptonedu — APPROVED** [review]
Reviewed

**Pull request merged**